tests/Foundatio.Tests/Locks/InMemoryLockTests.cs

```diff
@@ -21,12 +21,12 @@ public InMemoryLockTests(ITestOutputHelper output) : base(output)
 
     protected override ILockProvider GetThrottlingLockProvider(int maxHits, TimeSpan period)
     {
-        return new ThrottlingLockProvider(_cache, maxHits, period, Log);
+        return new ThrottlingLockProvider(_cache, maxHits, period, null, Log);
```

Comment:
I've seen this in quite a few spots, seems like we should be passing the time provider in from the cache?

Reply:
It gets it off the cache if it's available